<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Home">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="header" stretch="1,0,2,0">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="tbtnclear_search">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="elsearch"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnsearch">
              <property name="text">
               <string>search</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="home_tabs">
            <property name="tabPosition">
             <enum>QTabWidget::West</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="tabBarAutoHide">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="movies">
             <attribute name="title">
              <string>Movies</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QListWidget" name="movies_listwidget">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::NoDragDrop</enum>
                </property>
                <property name="defaultDropAction">
                 <enum>Qt::IgnoreAction</enum>
                </property>
                <property name="textElideMode">
                 <enum>Qt::ElideLeft</enum>
                </property>
                <property name="movement">
                 <enum>QListView::Static</enum>
                </property>
                <property name="isWrapping" stdset="0">
                 <bool>true</bool>
                </property>
                <property name="resizeMode">
                 <enum>QListView::Adjust</enum>
                </property>
                <property name="layoutMode">
                 <enum>QListView::Batched</enum>
                </property>
                <property name="viewMode">
                 <enum>QListView::IconMode</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="series">
             <attribute name="title">
              <string>Series</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QListWidget" name="series_listwidget">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="defaultDropAction">
                 <enum>Qt::IgnoreAction</enum>
                </property>
                <property name="textElideMode">
                 <enum>Qt::ElideLeft</enum>
                </property>
                <property name="movement">
                 <enum>QListView::Static</enum>
                </property>
                <property name="isWrapping" stdset="0">
                 <bool>true</bool>
                </property>
                <property name="resizeMode">
                 <enum>QListView::Adjust</enum>
                </property>
                <property name="layoutMode">
                 <enum>QListView::Batched</enum>
                </property>
                <property name="viewMode">
                 <enum>QListView::IconMode</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Watch">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QToolButton" name="btnhome">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="lposter">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="midLineWidth">
                 <number>1</number>
                </property>
                <property name="text">
                 <string>Poster</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_8"/>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="lenname">
                        <property name="font">
                         <font>
                          <family>Simplified Arabic Fixed</family>
                          <pointsize>16</pointsize>
                          <weight>50</weight>
                          <bold>false</bold>
                         </font>
                        </property>
                        <property name="text">
                         <string>EnName</string>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="lyear">
                        <property name="text">
                         <string>Year</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QLabel" name="larname">
                      <property name="text">
                       <string>ArName</string>
                      </property>
                      <property name="margin">
                       <number>-10</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="Line" name="line">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="lstars">
                    <property name="text">
                     <string>stars</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="font">
                     <font>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>•</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="linfo">
                    <property name="text">
                     <string>info</string>
                    </property>
                    <property name="textFormat">
                     <enum>Qt::MarkdownText</enum>
                    </property>
                    <property name="openExternalLinks">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="font">
                     <font>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>•</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lcate">
                    <property name="text">
                     <string>catego</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QScrollArea" name="scrollArea">
                    <property name="widgetResizable">
                     <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaWidgetContents_2">
                     <property name="geometry">
                      <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>714</width>
                       <height>95</height>
                      </rect>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_16">
                      <item>
                       <widget class="QLabel" name="lstory">
                        <property name="cursor">
                         <cursorShape>IBeamCursor</cursorShape>
                        </property>
                        <property name="text">
                         <string>story</string>
                        </property>
                        <property name="scaledContents">
                         <bool>true</bool>
                        </property>
                        <property name="wordWrap">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QTabWidget" name="tbepisodes">
                    <property name="currentIndex">
                     <number>-1</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <spacer name="verticalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <item>
                       <spacer name="horizontalSpacer_4">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>40</width>
                          <height>20</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_14">
                        <item>
                         <widget class="QPushButton" name="btnaddsub">
                          <property name="text">
                           <string>Add Externel Subtitle</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_10">
                          <item>
                           <widget class="QComboBox" name="coqual"/>
                          </item>
                          <item>
                           <widget class="QComboBox" name="cosubs"/>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_11">
                          <item>
                           <widget class="QPushButton" name="btnplay">
                            <property name="text">
                             <string>play</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QPushButton" name="btncloseplayer">
                            <property name="text">
                             <string>close player</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
